ansible/group_vars/all/vars

```diff
@@ -9,15 +9,23 @@ testenv_block:
   portgroup_mgmt: "{{ testenv_mgmt_network }}"
   portgroup_contrail: "{{ testenv_data_network }}"
   vms:
-    controller:
+    - name: "{{ testenv_name }}-controller"
+      type: controller
+      groups: controller
       template: "{{ testenv_controller_vmware_template }}"
-      nodes:
-      - name: "{{ testenv_name }}-controller"
-    wintb:
+      ip: 172.16.0.1
+      netmask: 255.255.0.0
+    - name: "{{ testenv_name }}-wintb01"
+      type: wintb
+      groups: testbed,windows
+      template: "{{ testenv_testbed_vmware_template }}"
+      ip: 172.16.0.2
+      netmask: 255.255.0.0
+      provision_options: "--vm-username '{{ windows_customization_user }}' --vm-password '{{ windows_customization_pass }}'"
```

Comment:
Please split this `provision_options` variable into two variables `vm-username` and `vm-password`, they should be optional.